gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -201,9 +208,19 @@ def format_query(self, query):
 
     Formatting logic is as follows:
     - Concatenates ``query`` parts into one string.
+    - Replaces macro with the corresponding value defined in _QueryMacros.
     - Replaces TABLE_NAME with the table associated for the query.
     """
-    return (' ').join(query).format(TABLE_NAME=self._table_name)
+    return self._replace_variables(self._replace_macros((' ').join(query)))
```

Comment:
nit: do you need the brackets around `' '`? i.e. does `..._macros(' '.join(query)))` work?

Reply:
Done.